### PR TITLE
ORLP-185.Fixed ability to add same deck to the course more than once

### PR DESCRIPTION
**Pull request by IhorF**
Fixed ability to add same deck to the course more than once.

### CODE REVIEW

File: src/main/java/com/softserve/academy/spaced/repetition/service/CourseService.java [hunk 1]
```diff
@@ -151,12 +152,14 @@ public void deleteLocalCourse(Long course_id) throws NotAuthorisedUserException
...
             }
         }
-
         userRepository.save(user);
     }
 
-    public void addDeckToCourse(Long course_id, Long deck_id) {
+    public void addDeckToCourse(Long course_id, Long deck_id) throws SameDeckInCourseException {
         Course course = courseRepository.findOne(course_id);
+        if(course.getDecks().stream().anyMatch(deck -> deck.getId()==deck_id)){
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/service/CourseService.java]: - As  both deck.getId() and deck_id are both objects, not primitives, equal() should be used.

- Please rename variables according to https://sbforge.org/sonar/rules/show/squid:S00117

Comment (zadorovskyi) [reply src/main/java/com/softserve/academy/spaced/repetition/service/CourseService.java]: deck.getId() return long....is that an object?

Comment (zadorovskyi) [reply src/main/java/com/softserve/academy/spaced/repetition/service/CourseService.java]: Sorry...it return Long))

Comment (antonbakalets) [reply src/main/java/com/softserve/academy/spaced/repetition/service/CourseService.java]: Ok. Just check this code:

Long v = null;
boolean b = 1L == v;

**Comment by b0hdan** [comment]
Everything seems to be okay here now.

**Pull request merged**